### PR TITLE
Fix typos in method names and javadoc

### PR DESCRIPTION
**Pull request by cmabdullah**
Fix typos in method names and javadoc; update the documentation to ensure it is more understandable and coherent.

### CODE REVIEW

File: spring-boot-project/spring-boot-docs/src/docs/antora/modules/reference/pages/testing/testcontainers.adoc [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 The https://www.testcontainers.org/[Testcontainers] library provides a way to manage services running inside Docker containers.
 It integrates with JUnit, allowing you to write a test class that can start up a container before any of the tests run.
-Testcontainers is especially useful for writing integration tests that talk to a real backend service such as MySQL, MongoDB, Cassandra and others.
+Testcontainers are especially useful for writing integration tests that talk to a real backend service such as MySQL, MongoDB, Cassandra and others.
```
Comment (wilkinsona) [inline spring-boot-project/spring-boot-docs/src/docs/antora/modules/reference/pages/testing/testcontainers.adoc:6]: This is referring to the Testcontainers open source project. It should be singular.

File: spring-boot-project/spring-boot-test/src/main/java/org/springframework/boot/test/context/SpringBootContextLoader.java [hunk 2]
```diff
@@ -141,7 +141,7 @@ private void assertHasClassesOrLocations(MergedContextConfiguration mergedConfig
 		boolean hasClasses = !ObjectUtils.isEmpty(mergedConfig.getClasses());
 		boolean hasLocations = !ObjectUtils.isEmpty(mergedConfig.getLocations());
 		Assert.state(hasClasses || hasLocations,
-				() -> "No configuration classes or locations found in @SpringApplicationConfiguration. "
+				() -> "No configuration classes or locations are found in @SpringApplicationConfiguration. "
```
Comment (wilkinsona) [inline spring-boot-project/spring-boot-test/src/main/java/org/springframework/boot/test/context/SpringBootContextLoader.java:144]: I don't think "are" is needed here.

**Review by wilkinsona — CHANGES_REQUESTED** [review]

File: spring-boot-project/spring-boot-test/src/main/java/org/springframework/boot/test/context/SpringBootContextLoader.java (hunk 2 above)
Comment (abdelhak-zaaim) [inline spring-boot-project/spring-boot-test/src/main/java/org/springframework/boot/test/context/SpringBootContextLoader.java:144]: Adding "are" doesn't significantly improve clarity

**Review by abdelhak-zaaim — COMMENTED** [review]
**Unnecessary Change**: The original error message was already grammatically correct and clear. Adding "are" does not significantly enhance readability or understanding.
**No Functional Impact**: This change does not affect the functionality of the code in any way.

File: spring-boot-project/spring-boot-docs/src/docs/antora/modules/reference/pages/testing/testcontainers.adoc (hunk 1 above)
Comment (cmabdullah) [reply spring-boot-project/spring-boot-docs/src/docs/antora/modules/reference/pages/testing/testcontainers.adoc]: I have updated the code accordingly. Thank you for the feedback.

File: spring-boot-project/spring-boot-test/src/main/java/org/springframework/boot/test/context/SpringBootContextLoader.java (hunk 2 above)
Comment (cmabdullah) [reply spring-boot-project/spring-boot-test/src/main/java/org/springframework/boot/test/context/SpringBootContextLoader.java]: Changes have been made, thank you for the feedback as always.

**Review by wilkinsona — APPROVED** [review]
Thanks very much for the updates, @cmabdullah.

**Comment by mhalbritter** [comment]
Thanks!

**Pull request closed without merging**